packages/node_modules/@webex/react-container-activity-list/src/formatters/at-mention.js

```diff
@@ -53,7 +53,7 @@ function AtMentionComponent({
       className={classNames(styles.atMention)}
       // eslint-disable-reason content is generated from elsewhere in the app
       // eslint-disable-next-line react/no-danger
-      dangerouslySetInnerHTML={{__html: content}}
+      dangerouslySetInnerHTML={{__html: content.replace(/<a(?![^>]*target="_blank"|\s*href\s*=\s*["'](?:mailto:|tel:))([^>]*)>/gi, '<a target="_blank"$1>')}}
```

Comment:
This component is used when there is an @mention present in the message

Reply:
Where does @mentioned links take to today in the widgets? Should we have a test case for that one as well?

Reply:
The @mentioned links aren't clickable. They add a new tag called <spark-mention /> and wrap the person data around it. Whenever a mention is present in a message, this component gets used instead of react-component-activity-text. So I had to add the replacement logic here as well.
eg:
`<spark-mention data-object-type="person" data-object-id="801bb984-343b-4f6b-97ae-d13c91d4b877">John Doe</spark-mention>`